src/main/java/net/sf/rails/game/PlayerManager.java

```diff
@@ -281,11 +281,7 @@ public Player reorderPlayersByCash (boolean ascending) {
 
        final boolean ascending_f = ascending;
 
-       Comparator<Player> cashComparator =  new Comparator<Player>() {
-           public int compare (Player p1, Player p2) {
-               return ascending_f ? p1.getCash() - p2.getCash() : p2.getCash() - p1.getCash();
-           }
-       };
+       Comparator<Player> cashComparator = (p1, p2) -> ascending_f ? p1.getCash() - p2.getCash() : p2.getCash() - p1.getCash();
```

Comment:
Use of a lambda to replace anonymous classes

Reply:
Here I would recommend to use:

```java
Comparator<Player> cashComparator = Comparator.comparing(Player::getCash);
```

If you need a comparator for the other direction you can then use:

```java
cashComparator.reversed()
```

Reply:
interesting, when did that :: operater got implemented ?

Reply:
@neutronc it was added as in 8 to help with `stream()` and lambdas and is a reference to a method, static, object, etc. Check section 3 here: https://www.baeldung.com/java-8-new-features